setup.py

```diff
@@ -19,22 +19,24 @@
 import sys
 import tokenize
 
-from distutils.command.clean import clean
-from distutils.command.build_scripts import build_scripts
-from distutils.spawn import find_executable
-from platform import system, processor
-from setuptools import find_packages, setup
-from shutil import rmtree
-from subprocess import CalledProcessError, check_call, check_output
+from distutils import sysconfig
+from setuptools import Extension, find_packages, setup
+from subprocess import CalledProcessError, check_output
 
 DEFAULT_SHELL_ENCODING = "ascii"
 LDFLAGS_LIBRARY_SWITCH = "-l"
 LDFLAGS_LIBRARY_PATH_SWITCH = "-L"
 CFLAGS_INCLUDE_PATH_SWITCH = "-I"
 
-if (sys.version_info[0] == 2 and sys.version_info[1] < 7) or \
-   (sys.version_info[0] == 3 and sys.version_info[1] < 4):
-    print("Genomedata requires Python version 2.7 or 3.4 or later")
+MINIMUM_PYTHON_VERSION_MAJ = 3
```

Comment:
I think this can all be replaced by putting `python_requires='>=3.7'` as an argument to `setup()`.

Reply:
Yes you are correct. It's actually already there so this section is redundant. Will remove